mule-migration-tool-e2e-tests/src/test/java/com/mulesoft/tools/migration/e2e/AbstractEndToEndTestCase.java

```diff
@@ -150,7 +157,10 @@ private void comparePom(Path expectedPath, Path migratedPath) {
       StringWriter expectedOutput = new StringWriter();
       StringWriter migratedOutput = new StringWriter();
       mavenWriter.write(expectedOutput, new PomModelBuilder().withPom(expectedPath).build().getMavenModelCopy());
-      mavenWriter.write(migratedOutput, new PomModelBuilder().withPom(migratedPath).build().getMavenModelCopy());
+      PomModel migratedPomModel = new PomModelBuilder().withPom(migratedPath).build();
```

Comment:
I think is better to run the version normalizer also on the expectedPom so there's no need to replace version numbers with the `VERSION` placeholder and thus is easier for people to contribute tests from real apps